lasagne/layers/conv.py

```diff
@@ -181,12 +175,12 @@ class Conv1DLayer(Layer):
     Notes
     -----
     Theano's default convolution function (`theano.tensor.nnet.conv.conv2d`)
```

Comment:
Hmm, sort of. For `pad='same'` and `stride=(1,)`, it crops a full convolution. For `pad='same'` and `stride!=(1,)` or for custom padding, it explicitly pads the input. I can change the warning to:
```
Theano's underlying convolution (:func:`theano.tensor.nnet.conv.conv2d`)
only supports ``pad=0`` and ``pad='full'``. This layer emulates other modes
by cropping a full convolution or explicitly padding the input with zeroes.
```